src/libs/UserUtils.ts

```diff
@@ -104,7 +104,7 @@ function getDefaultAvatarURL(accountID: string | number = '', isNewDot = false):
  * Given a user's avatar path, returns true if user doesn't have an avatar or if URL points to a default avatar
  * @param [avatarURL] - the avatar source from user's personalDetails
  */
-function isDefaultAvatar(avatarURL?: string): boolean {
+function isDefaultAvatar(avatarURL?: React.FC<SvgProps> | string): boolean {
```

Comment:
I think that renaming `avatarURL` to `avatarSource` would be more readable since this can be a react component 😄

Reply:
Makes sense! I've updated now